apollo-sample/src/main/java/com/example/apollographql/sample/SampleApplication.java

```diff
@@ -31,10 +32,12 @@
         .baseUrl(BASE_URL)
         .client(okHttpClient)
         .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
-        .addConverterFactory(new ApolloConverterFactory.Builder().build())
+        .addConverterFactory(new ApolloConverterFactory.Builder()
+            .withResponseFieldMapper(FeedQuery.Data.class, new FeedQuery.Data.Mapper(FeedQuery.Data.FACTORY))
```

Comment:
so sorry could you explain this line again.  Will we need to register every object or is this for a particular custom field